tests/Unit/Helpers/DefinitionValidatorTest.php

```diff
@@ -98,23 +119,23 @@ public function testInvalidMethodCalls(array $methodCalls, string $message): voi
     public function dataErrorOnPropertyOrMethodTypo(): array
     {
         return [
-            ['dev', true, '~^Invalid definition: key "dev" is not allowed\. Did you mean "dev\(\)" or "\$dev"\?$~'],
-            ['setId', [42], '~^Invalid definition: key "setId" is not allowed\. Did you mean "setId\(\)" or "\$setId"\?$~'],
-            ['()test', [42], '~^Invalid definition: key "\(\)test" is not allowed\. Did you mean "test\(\)" or "\$test"\?$~'],
-            ['var$', true, '~^Invalid definition: key "var\$" is not allowed\. Did you mean "var\(\)" or "\$var"\?$~'],
-            [' var$', true, '~^Invalid definition: key " var\$" is not allowed\. Did you mean "var\(\)" or "\$var"\?$~'],
-            ['100$', true, '~^Invalid definition: key "100\$" is not allowed\.$~'],
-            ['test-тест', true, '~^Invalid definition: key "test-тест" is not allowed\.$~'],
+            ['dev', true, 'Invalid definition: key "dev" is not allowed. Did you mean "dev()" or "$dev"?'],
+            ['setId', [42], 'Invalid definition: key "setId" is not allowed. Did you mean "setId()" or "$setId"?'],
+            ['()test', [42], 'Invalid definition: key "()test" is not allowed. Did you mean "test()" or "$test"?'],
+            ['var$', true, 'Invalid definition: key "var$" is not allowed. Did you mean "var()" or "$var"?'],
+            [' var$', true, 'Invalid definition: key " var$" is not allowed. Did you mean "var()" or "$var"?'],
+            ['100$', true, 'Invalid definition: key "100$" is not allowed.'],
+            ['test-тест', true, 'Invalid definition: key "test-тест" is not allowed.'],
```

Comment:
Why this change?

After this change added mutants.

Reply:
There is no need in regular expressions

Reply:
Need check message full. Without regex pass message with any suffix. 

See mutants.